udata/api/oauth2.py

```diff
@@ -79,6 +81,9 @@ class OAuth2Client(ClientMixin, db.Datetimed, db.Document):
         'collection': 'oauth2_client'
     }
 
+    def get_client_id(self):
```

Comment:
`client_id` property below not good enough?

Reply:
We have to implement this method, the PKCE flow requires it. An execption `NotImplemented` is raised otherwise.

Reply:
use `get_client_id` in `client_id` then.